docs/design/autoware-architecture/localization/index.md

```diff
@@ -1,5 +1,294 @@
+- The environment has laser beams of the same frequency as LiDAR
+
+#### Functionalities
+
+- The system can estimate the vehicle location on the point cloud map with the error of ~10cm
+- The system is Operable at the night
+
+### 3D-LiDAR or Camera + Vector Map
+
+#### Expected situation
+
+- Road with clear white lines and loose curvatures, such as highway or ordinary local road
+
+#### Situation that can make the system unstable
+
+- White lines are scratchy or covered by rain or snow
+- Tight curvature such as intersections
+- Large reflection change of the road surface caused by rain or paint
+
+#### Functionalities
+
+- Correct vehicle positions along the lateral direction
+- Pose correction along the longitudinal can be inaccurate. But we can resolve this by fusing with GNSS
+
+### GNSS
+
+#### Expected situation
+
+- The vehicle is placed in an open environment with no or few surrounding objects, such as a rural landscape
+
+#### Situation that can make the system unstable
+
+- GNSS signals are intercepted by surrounding objects, e.g., tunnels or buildings
+
+#### Functionalities
+
+- The system can estimate the vehicle position with an error of ~10m in the world coordinate
+- With RTK attached, the accuracy can be improved to ~10cm
+- The system with this configuration can work without environment maps
+
+### Camera (Visual Odometry, Visual SLAM)
+
+#### Expected situation
+
+- The vehicle is placed in a texture-rich environment, such as urban areas
+
+#### Situation that can make the system unstable
+
+- The vehicle is placed in a Texture-less environment
+- The vehicle is surrounded by other objects
+- The camera observes significant illumination changes such as sunshine, headlight, or brightness change at the exit of a tunnel
+- The vehicle is placed in a dark environment
+
+#### Functionalities
+
+- The system can estimate the odometry by tracking visual features
+
+### Wheel speed sensor
+
+#### Expected situation
+
+- The vehicle is running on an ordinary road
+
+#### Situation that can make the system unstable
+
+- The vehicle is running on a slippy or bumpy road
+- The vehicle cannot observe the correct wheel speed because of poor road surface conditions such as bumps or slips
+
+#### Functionalities
+
+- The system can acquire the vehicle velocity and estimate the traveled distance
+- The system can estimate more accurate poses in a higher frequency by fusing with external sensors
+
+<!-- NOTE: Is the second item necessary? Because it is general for any other sensors -->
+
+### IMU
+
+#### Expected environments
+
+- Ordinary roads
+
+<!-- NOTE: The availability of IMU is highest among the sensors attached to vehicles. Therefore we should list more situations or say we can use the IMU almost anywhere -->
+
+#### Situation that can make the system unstable
+
+- IMUs have a bias that is dependent on the surrounding temperature. This can cause incorrect sensor observation or odometry drift
```

Comment:
The bias is a little bit complicated. The detailed explanation is written in the BIAS section of this page and especially what we need to care about the most is the turn-on bias.

https://www.vectornav.com/resources/inertial-navigation-primer/specifications--and--error-budgets/specs-imuspecs

Usually the bias gradually changes as the surrounding temperature changes. I think this is because IMU is a semiconductor, which changes its characteristics by the surrounding temperature, like a transistor.

Reply:
Would it be an idea to add that link to the page above if anyone wants to get a better idea about bias?

Reply:
Alternatively, perhaps we need to consider adding footnotes to the bottom of design docs, so that relevant information can be included but in such a way to avoid affecting the flow of the main content.

Reply:
@kenji-miyake Is it ok to include an external link to the design doc page? This seems a link to a device production company

Reply:
@LalithVipulananthan Yes! Footnote is a great idea!

Reply:
I think it's okay as it seems to be a link to general information.

Reply:
@LalithVipulananthan Mm? It's already enabled.
https://github.com/autowarefoundation/autoware-documentation/blob/533f0fc640541d468039d3d0d5796dec291925e8/mkdocs.yaml#L66

Reply:
Oops. 😅 

@IshitaTakeshi I have updated my suggestion above to add the information about IMU bias as a footnote.